pkg/cluster/node.go

```diff
@@ -53,20 +53,23 @@ type NodeInfo struct {
 
 //GetNodeExternalIpList gets the list of External Node IPs of kubeslice-gateway nodes
 
-func (n *NodeInfo) getNodeExternalIpList() ([]string, error) {
+func (n *NodeInfo) getNodeExternalIpList(networkMode bool) ([]string, error) {
```

Comment:
networkMode seems to be little confusing, isNetworkPresent as a boolean can be more meaningful